src/wam2layers/tracking/backtrack.py

```diff
@@ -203,33 +202,3 @@ def run_experiment(config_file):
             write_output(output, t1, config)
 
     logger.info("Experiment complete.")
-
-
-###########################################################################
-# The code below makes it possible to run wam2layers from the command line:
-# >>> python backtrack.py path/to/cases/era5_2021.yaml
-# or even:
-# >>> wam2layers backtrack path/to/cases/era5_2021.yaml
-###########################################################################
-
-
-@click.command()
-@click.argument("config_file", type=click.Path(exists=True))
-def cli(config_file):
-    """Run WAM2layers backtrack experiment from the command line.
-
-    CONFIG_FILE: Path to WAM2layers experiment configuration file.
-
-    Usage examples:
-
-        \b
-        - python path/to/backtrack.py path/to/cases/era5_2021.yaml
-        - wam2layers backtrack path/to/cases/era5_2021.yaml
-    """
-    logger.info("Welcome to WAM2layers.")
-    logger.info("Starting backtrack experiment.")
```

Comment:
I liked this welcome message. Can we keep it and make this consistent also for the other CLI commands?

Reply:
Yes, now we have this welcome message in all cli commands.